File: sutom-front-end/sutom-front-end/src/app/models/guessResult.ts
export class LetterResult {
    constructor(public letter: string = '') { }
    public status: 'correct' | 'misplaced' | 'wrong' | 'empty' = 'empty';
}

export interface GuessResult {
    correct: boolean;
    showInfoMessage: boolean;
    letterResults: LetterResult[];
}

File: Sutom/Sutom.Domain/Entites/GuessResult.cs
namespace Sutom.Application.Models
{
    public class LetterResult
    {
        public char Letter { get; set; }
        public string Status { get; set; } = string.Empty;
    }


    public class GuessResult
    {
        public bool Correct { get; set; }
        public bool ShowInfoMessage { get; set; }
        public List<LetterResult> LetterResults { get; set; } = new List<LetterResult>();

    }
}
